spring-kafka/src/main/java/org/springframework/kafka/event/ConsumerPartitionPausedEvent.java

```diff
@@ -47,13 +48,13 @@ public ConsumerPartitionPausedEvent(Object source, Object container, TopicPartit
 	 * Return the paused partition.
 	 * @return the partition.
 	 */
-	public TopicPartition getPartitions() {
+	public TopicPartition getPartition() {
```

Comment:
Since this is a public API, it might be better to deprecate this method, mark it for removal, and add the new one `getPartition()`. In a future version, we can completely remove this.

Reply:
Thanks for your comment! I just [made a change](https://github.com/spring-projects/spring-kafka/pull/3293/commits/804ae8501d20de71a211faffa5ddf5f7b65b3e83) and check it please.